### PR TITLE
Feature/update java code after supporting def params

### PR DESCRIPTION
**Pull request by Saamyyy**
## what 
- update the java code after supporting the default params in java 
- add pro guard rules  for the sdk 
- update the sdk versions to latest

### CODE REVIEW

File: mobile-sdk/android/customer-secret/java/create-customer-secret.java [hunk 1]
```diff
@@ -7,10 +7,9 @@
         return Unit.INSTANCE;
     }));
 
-// On "Pay" tapped
+// On "Pay" clicked
```
Comment (monica-m-ps) [inline mobile-sdk/android/customer-secret/java/create-customer-secret.java:10]: This for mobile-only? Then "tapped" is apt.

File: mobile-sdk/android/customer-secret/java/create-customer-secret.java [hunk 2]
```diff
@@ -7,10 +7,9 @@
...
 dojoPaymentFlowHandler.startPaymentFlow(
     new DojoPaymentFlowParams(
         "<paymentIntentId>",
-        "<secret>",// Customer secret key
-        null
+        "<customer secret key>"
```
Comment (monica-m-ps) [inline mobile-sdk/android/customer-secret/java/create-customer-secret.java:14]: We follow the api response parameters like https://docs.dojo.tech/payments/api#tag/Customers/operation/Customers_CreateCustomerSecret

So <secret> and then a comment is preferred.

File: mobile-sdk/android/payment-intent/java/capture-card.java [hunk 3]
```diff
@@ -7,10 +7,8 @@
         return Unit.INSTANCE;
     }));
 
-// On "Pay" tapped
+// On "Pay" clicked
```
Comment (monica-m-ps) [inline mobile-sdk/android/payment-intent/java/capture-card.java:10]: Same comment across all these samples.

**Review by monica-m-ps — APPROVED** [review]

**Comment by monica-m-ps** [comment]
Adding @AleksandrMalafeev , @RomanKondratyev , @deniss-kaibagarovs for review for whoever approves first.

**Review by deniss-kaibagarovs — APPROVED** [review]

**Pull request merged**